metricbeat/_meta/beat.full.yml

```diff
@@ -121,6 +121,14 @@ metricbeat.modules:
   #period: 10s
   #hosts: ["tcp://127.0.0.1:14567"]
 
+#------------------------------- Jolokia Module ------------------------------
+- module: jolokia
+  metricsets: ["dynamic"]
+  enabled: true
+  period: 10s
+  hosts: ["localhost"]
+  namespace: "jolokia_metrics"
```

Comment:
lets call this `metrics` as `jolokia` will be already part of the namespace anyways.